docs/notebook_version_standardizer.py

```diff
@@ -45,6 +45,40 @@ def _standardize_versions(notebooks, desired_version="3.8.6"):
         _standardize_python_version(notebook, desired_version)
 
 
+def _check_execution_and_output(notebook):
+    with open(notebook, "r") as f:
+        source = json.load(f)
+        for cells in source["cells"]:
+            if cells["cell_type"] == "code" and cells["execution_count"] != None:
+                return False
+    return True
+
+
+def _fix_execution_and_output(notebook):
+    with open(notebook, "r") as f:
+        source = json.load(f)
+        for cells in source["cells"]:
+            if cells["cell_type"] == "code" and cells["execution_count"] != 'null':
```

Comment:
If we parse the file into json, don't the 'null's get convered to `None`?

Reply:
I think you're right! A typo leftover